.github/workflows/ci.yml

```
@@ -13,6 +13,7 @@ concurrency:
 jobs:
   puppet:
     name: Puppet
-    uses: voxpupuli/gha-puppet/.github/workflows/beaker.yml@v2
+    uses: evgeni/gha-puppet/.github/workflows/beaker.yml@vagrant
     with:
       pidfile_workaround: 'false'
+      beaker_hypervisor: vagrant
```

Should this be libvirt?
```suggestion
      beaker_hypervisor: vagrant_libvirt
```

---

No. The Beaker hypervisor is Vagrant. Which hypervisor vagrant then uses is a different beast. ;)

(hypervisors all the way down)

---

(and I won't go down the rabbithole that libvirt can use different backends, besides qemu)

---

That's surprising, because I have:
```console
$ env | grep BEAKER
BEAKER_HYPERVISOR=vagrant_libvirt
```